TimeSliceNet/TimeSlice/TimeDependentParentChildCollection.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text.Json.Serialization;
+
+namespace TimeSlice
+{
+    /// <summary>
+    ///     Describes the possible kinds of time dependent relationships between parents and children
+    /// </summary>
+    public enum TimeDependentCollectionType
+    {
+        /// <summary>
+        ///     &lt;= 1 child assigned to a parent at a time
+        /// </summary>
+        AllowOverlaps,
+
+        /// <summary>
+        ///     prevent overlapping time slices (there might be &gt;1 child assigned to a parent at a time
```

Comment:
Closing ) is missing in the comment ;)

Reply:
#15 sorry, already deleted the branch. wasn't sure if you had time to look into it... but only came to the conclusion 2 minutes fter I requested your review :P